intellij/src/saros/intellij/ui/wizards/AddProjectToSessionWizard.java

```diff
@@ -221,6 +228,11 @@ private void doNewModule(
             return;
           }
 
+          sessionManager
+              .getSession()
```

Comment:
NPE number 1

Reply:
You can easily proof it, just disconnect (pull the plug). It is null afterwards.

https://github.com/saros-project/saros/blob/master/core/src/saros/session/SarosSessionManager.java#L368